src/pf-tooltip/index.html

```diff
@@ -32,21 +32,21 @@
     </div>
 
     <hr>
-    <div id="div">
+    <div class="tooltip-wrapper-div">
```

Comment:
@cdcabrera yes, HTML ids are typically unique. I changed IDs to classes and gave them a different name. Does this work?